From 9614277f759768cfc156a3493dd2b60fae7d63d5 Mon Sep 17 00:00:00 2001
From: Mike Maietta <mmaietta@users.noreply.github.com>
Date: Sun, 18 Jan 2026 15:06:36 -0800
Subject: [PATCH] Ensure text size declarations are honoured in JSON
 configurations (#276)

---
 src/dmgbuild/core.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/dmgbuild/core.py b/src/dmgbuild/core.py
index d8ea6d3..bf1ee56 100644
--- a/src/dmgbuild/core.py
+++ b/src/dmgbuild/core.py
@@ -100,6 +100,9 @@ def load_json(filename, settings):
     if bk is not None:
         settings["background"] = bk
     settings["icon_size"] = json_data.get("icon-size", 80)
+    tsize = json_data.get("text-size", None)
+    if tsize is not None:
+        settings["text_size"] = tsize
     wnd = json_data.get(
         "window",
         {
-- 
2.50.1 (Apple Git-155)

